langstream-e2e-tests/pom.xml

```diff
@@ -52,6 +52,12 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>langstream-kafka-runtime</artifactId>
```

Comment:
please note that this is adding the kafka client dependency and also Kafka Connect with tons of third party deps (like versions of Netty...)